.github/workflows/go-licenses.yml

```diff
@@ -0,0 +1,62 @@
+name: go-licenses
+
+on:
+  # run action when a change lands in the main branch which updates go.mod or
+  # our license template file. Also allow manual triggering.
+  push:
+    branches:
+      - main
+    paths:
+      - go.mod
+      - .github/licenses.tmpl
+      - .github/workflows/go-licenses.yml
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-$${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  tailscale:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: go.mod
+
+      - name: Install go-licenses
+        run: |
+          go install github.com/google/go-licenses@v1.2.2-0.20220825154955-5eedde1c6584
+
+      - name: Run go-licenses
+        env:
+          # include all build tags to include platform-specific dependencies
+          GOFLAGS: "-tags=cgo,darwin,freebsd,ios,js,linux,openbsd,wasm,windows"
```

Comment:
Do you have thoughts on how to accommodate Android?

Windows pulls in some additional dependencies for its UI, like lxn, will we handle those from here later or some other way?

Reply:
oops, forgot to include android here!  I'll add that.

lxn should show up in the generated list for windows which will run as a similar action inside corp.  For non-go dependencies, we'll simply include them directly in the respective template file and maintain those lists manually (which shouldn't be too bad since they don't change often)